core/templates/dev/head/pages/admin/admin.mainpage.html

```diff
@@ -78,14 +95,8 @@
     <script src="/templates/dev/head/filters/TruncateFilter.js"></script>
     <script src="/templates/dev/head/filters/UnderscoresToCamelCaseFilter.js"></script>
 
-    <script src="/templates/dev/head/pages/admin/Admin.js"></script>
-    <script src="/templates/dev/head/pages/admin/AdminNavbarDirective.js"></script>
     <script src="/templates/dev/head/pages/admin/AdminRouterService.js"></script>
     <script src="/templates/dev/head/pages/admin/AdminTaskManagerService.js"></script>
-    <script src="/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js"></script>
-    <script src="/templates/dev/head/pages/admin/activities_tab/AdminProdModeActivitiesTabDirective.js"></script>
-    <script src="/templates/dev/head/pages/admin/config_tab/AdminConfigTabDirective.js"></script>
-    <script src="/templates/dev/head/pages/admin/jobs_tab/AdminJobsTabDirective.js"></script>
 
     <script src="/templates/dev/head/pages/admin/roles_tab/AdminRolesTabDirective.js"></script>
```

Comment:
I think you will need to remove this as well. The e2e test are failing due to this.